blocks/init/custom/carousel/assets/index.js

```diff
@@ -0,0 +1,10 @@
+$(function() {
```

Comment:
We can use
https://developer.wordpress.org/block-editor/packages/packages-dom-ready/

Reply:
I have added an issue for this

Reply:
https://github.com/infinum/eightshift-frontend-libs/issues/41